utils.py

```diff
@@ -1,24 +1,30 @@
-
+import os
+user_name = os.environ.get('USER')
 
 browsers_definition = {
   "Chrome": {
     "executable": "Google Chrome",
+    "process_name": "Google Chrome",
     "identifier": "com.google.Chrome"
   },
   "Canary": {
     "executable": "Google Chrome Canary",
+    "process_name": "Google Chrome Canary",
     "identifier": "com.google.Chrome.canary"
   },
   "Chromium": {
-    "executable": "Chromium",
+    "executable": f"/Users/{user_name}/git/chromium/src/out/Release/Chromium.app/Contents/MacOS/Chromium",
```

Comment:
The part "Users/{user_name}/git/chromium/src/out/Release" may vary (especially with multiple builds). Do you think we can make this an argument of ScenarioConfig or something?

Reply:
Good point, I made it enforced in ScenarioConfig but passed in through a command line argument.